events/api.py

```diff
@@ -1840,16 +1841,48 @@ class RegistrationSerializer(LinkedEventsSerializer, RegistrationBaseSerializer)
         queryset=Event.objects.all(),
     )
 
+    only_admin_visible_fields = ("created_by", "last_modified_by", "registration_users")
+
+    def _create_registration_users(self, registration, registration_users):
+        for registration_user in registration_users:
+            [user_instance, created] = RegistrationUser.objects.get_or_create(
```

Comment:
I'd remove the brackets
```suggestion
            user_instance, created = RegistrationUser.objects.get_or_create(
```

Reply:
Fixed in https://github.com/City-of-Helsinki/linkedevents/pull/631/commits/22a53edd3def57c8004e3b1787b9c0942d3b1f1f